api/src/app.module.ts

```diff
@@ -7,9 +7,10 @@ import { LinksModule } from './links/links.module';
 import { MongooseModule } from  '@nestjs/mongoose';
 import { HttpErrorFilter } from './shared/http-error.filter';
 // import { HeaderModule } from './header/header.module';
+import { EstimateModule } from './estimate/estimate.module';
 
 @Module({
-  imports: [LinksModule, MongooseModule.forRoot(process.env.MONGO_URI)],
+  imports: [LinksModule, MongooseModule.forRoot(process.env.MONGO_URI), EstimateModule],
```

Comment:
better to be consistent here, keep the non-API related module either in the beginning or the ned of the imports.

Reply:
it's an auto import issue
noticed